modules/hcp-must-gather-day-2.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-must-gather-day-2_{context}"]
+= Gathering {product-title} data for a hosted cluster
+
+You can gather {product-title} debugging information for a hosted cluster by using the {mce-short} web console or by using the `hcp` CLI.
+
+.Prerequisites
+
+* You must have `cluster-admin` access to the management cluster.
+
+* You need the `name` value for the `HostedCluster` resource and the namespace where the CR is deployed.
+
+* You must have the `hcp` command-line interface installed. For more information, see "Installing the {hcp} command-line interface".
+
+* You must have the OpenShift CLI (`oc`) installed.
+
+* You must ensure that the `kubeconfig` file is loaded and is pointing to the management cluster.
+
+.Procedure
+
+* To collect must-gather information by using the web console, complete the following steps:
++
+. In the web console, select *All Clusters* and select the cluster you want to troubleshoot.
+. In the upper-right corner, select *Download kubeconfig*.
+. Export the downloaded `kubeconfig` file.
+. Collect the must-gather information by entering the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather
+----
+
+* To collect must-gather information by using the `hcp` CLI, complete the following steps:
```

Comment:
I would remove `hcp` since the following steps run both `hcp` and `oc` commands.

Reply:
Will do